apps/rush-lib/src/cli/taskRunner/ProjectTask.ts

```diff
@@ -63,15 +63,13 @@ export class ProjectTask implements ITaskDefinition {
   }
 
   public execute(writer: ITaskWriter): Promise<TaskStatus> {
-    return new Promise<TaskStatus>((resolve: (status: TaskStatus) => void, reject: (errors: TaskError[]) => void) => {
-      try {
-        const taskCommand: string = this._getScriptToRun();
-        const deps: IPackageDependencies | undefined = this._getPackageDependencies(taskCommand, writer);
-        this._executeTask(taskCommand, writer, deps, resolve, reject);
-      } catch (error) {
-        reject([new TaskError('executing', error.toString())]);
-      }
-    });
+    try {
+      const taskCommand: string = this._getScriptToRun();
+      const deps: IPackageDependencies | undefined = this._getPackageDependencies(taskCommand, writer);
+      return this._executeTask(taskCommand, writer, deps);
+    } catch (error) {
+      return Promise.reject([new TaskError('executing', error.toString())]);
```

Comment:
>toString [](start = 62, length = 8)

Use `error.message` instead of `error.toString()`.  The latter form adds a "Error:" prefix that typically interferes with the regular formatting